### PR TITLE
show the inset group style in fluent demo app

### PR DESCRIPTION
**Pull request by harrieshin**
### Platforms Impacted
- [x] iOS
- [ ] macOS

### Description of changes

Binary change:

| File | Before | After | Delta |
|------|--------|-------|-------|
|na | na| na| na|

(a summary of the changes made, often organized by file)

### Verification

(how the change was tested, including both manual and automated tests)

| Before                                       | After                                      |
|----------------------------------------------|--------------------------------------------|
| image |  image |
| image | image |
|  image| image |
| image| image|
| image| image|
| image| image|

### Pull request checklist

This PR has considered:
- [ ] Light and Dark appearances
- [ ] iOS supported versions (all major versions greater than or equal current target deployment version)
- [ ] VoiceOver and Keyboard Accessibility
- [ ] Internationalization and Right to Left layouts
- [ ] Different resolutions (1x, 2x, 3x)
- [ ] Size classes and window sizes (iPhone vs iPad, notched devices, multitasking, different window sizes, etc)
- [ ] iPad [Pointer interaction](https://developer.apple.com/documentation/uikit/pointer_interactions)
- [ ] [SwiftUI](https://developer.apple.com/tutorials/swiftui) consumption (validation or new demo scenarios needed)
- [ ] Objective-C exposure (provide it only if needed)
 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://api.github.com/repos/microsoft/fluentui-apple/pulls/1550)

### CODE REVIEW

File: ios/FluentUI.Demo/FluentUI.Demo/DemoListViewController.swift [hunk 1]
```diff
@@ -121,6 +140,13 @@ class DemoListViewController: DemoTableViewController {
     let cellReuseIdentifier: String = "TableViewCell"
     private static var isFirstLaunch: Bool = true
     private static let lastDemoControllerKey: String = "LastDemoController"
+    private let cellTypeButton: UIButton = {
+        let button = Button()
+        button.setTitle("Type", for: .normal)
+        return button
+    }()
```
Comment (huwilkes) [inline ios/FluentUI.Demo/FluentUI.Demo/DemoListViewController.swift:147]: What's the goal of adding this button? Is the title supposed to change to say the type?

Comment (harrieshin) [reply ios/FluentUI.Demo/FluentUI.Demo/DemoListViewController.swift:147]: users can switch between the plain and insetgroup. it launches the menu

File: ios/FluentUI.Demo/FluentUI.Demo/Demos/TableViewCellDemoController.swift [hunk 2]
```diff
@@ -11,7 +11,20 @@ import UIKit
 class TableViewCellDemoController: DemoTableViewController {
     let sections: [TableViewSampleData.Section] = TableViewCellSampleData.sections
 
-    private var isGrouped: Bool = false {
+    override init(nibName nibNameOrNil: String?, bundle nibBundleOrNil: Bundle?) {
```
Comment (laminesm) [inline ios/FluentUI.Demo/FluentUI.Demo/Demos/TableViewCellDemoController.swift:14]: Why do we need to add this if it's already set in DemoTableViewController?

Comment (harrieshin) [reply ios/FluentUI.Demo/FluentUI.Demo/Demos/TableViewCellDemoController.swift:14]: this is for the shimmertableviewcell. it isn't using the custom route, it uses the tableviewcelldemo directly.

**Review by laminesm — APPROVED** [review]

File: ios/FluentUI.Demo/FluentUI.Demo/DemoListViewController.swift (hunk 1 above)
Comment (huwilkes) [reply ios/FluentUI.Demo/FluentUI.Demo/DemoListViewController.swift:147]: Would it be possible to add it to the demo appearance controller instead? It seems odd to start adding customization options on the cell to launch the demo.

Comment (harrieshin) [reply ios/FluentUI.Demo/FluentUI.Demo/DemoListViewController.swift:147]: once you create a tableview you can't edit the style. so it has to be done before the controller is created

**Pull request merged**